### PR TITLE
Add namespace and team_id to all postgres related entities

### PR DESCRIPTION
**Pull request by avaczi**
In many cases either one or both of these were missing, causing confusion in rare cases (where names otherwise would clash, for example, especially with the database entities).

### CODE REVIEW

**Comment by FxKu** [comment]
:+1:

File: zmon_agent/discovery/kubernetes/cluster.py [hunk 1]
```diff
@@ -981,7 +980,7 @@ def get_postgresql_clusters(kube_client, cluster_id, alias, environment, region,
         else:
             ss = statefulset[0]
 
-        postgresql = [pg for pg in pgs if pg['name'] == version and pg['namespace'] == service_namespace]
+        postgresql = [pg for pg in postgresqls if pg['name'] == version and pg['namespace'] == service_namespace]
```
Comment (pitr) [inline zmon_agent/discovery/kubernetes/cluster.py:984]: what is the reason for this change?

Comment (avaczi) [reply zmon_agent/discovery/kubernetes/cluster.py:984]: It used to be there for some workaround when the discovery functions were still generators (don't exactly remember why it was needed, but there was a reason).  Now that this is not the case anymore, I removed 'materializing' to a list.

**Comment by vetinari** [comment]
:+1:

**Comment by pitr** [comment]
👍
